src/components/DuplicatesPage/MergedItem/MergedItem.js

```diff
@@ -0,0 +1,16 @@
+import Table from "../../Common/Table/Table";
+import * as React from "react";
+import {DUPLICATES_TABLE_HEADERS} from "../../../constants/duplicatesHeaders";
+import _ from "lodash";
+
+const MergedItem = ({duplicate}) => (
+    <Table headers={DUPLICATES_TABLE_HEADERS}>
+        <tr>
+            {_.values(duplicate).map((value, index) => (
+                <td key={'key-' + index}>{value}</td>
+            ))}
+        </tr>
+    </Table>
+);
+
```

Comment:
Decide if you use lodash or generic functions from JavaScript. Try to be consistent -> extract map to it's own method so it has semantic meaning